packages/haul-ram-bundle-webpack-plugin/package.json

```diff
@@ -18,5 +18,8 @@
     "webpack": "4.x",
     "webpack-sources": "1.x"
   },
-  "gitHead": "c41becc82dfeafb07b642ebed8f082775647ec3f"
+  "gitHead": "c41becc82dfeafb07b642ebed8f082775647ec3f",
+  "dependencies": {
+    "jest-worker": "^24.9.0"
```

Comment:
FYI, Jest 25 just came out, so you might want to be on the latest :) There is one breaking change to this package, but not sure if it affects you, see https://github.com/facebook/jest/blob/master/CHANGELOG.md